src/layouts/Generator.astro

```diff
@@ -0,0 +1,436 @@
+---
+import Page from "./Page.astro";
+
+const props = Astro.props;
+---
+<Page content={props.content} frontmatter={props.frontmatter}>
+    <noscript>
+        <article class="message is-danger">
+            <div class="message-body">
+                <p>Javascript is required to view this page</p>
+            </div>
+        </article>
+    </noscript>
+
+    <p>
+        This page provides the latest versions of Quilt and its components, ready to copy paste into your <code>libs.versions.toml</code> file.
+        It is located in the <code>gradle</code> folder if you are using our template mod.
+    </p>
```

Comment:
This text should probably be different than the one on the latest versions page?

Reply:
Fixed